### ISSUE TITLE
[Tongues/ML] Capstone: learn shared latent speech dynamics from audio + articulation without phone segmentation

### ISSUE THREAD

**Issue by dancxjo** [dancxjo/conduit#2145]
Parent: #2132
Depends on: #2133 #2134 #2135 #2137 #2138 #2139 #2140 #2141 #2142 #2143 #2144

## Outcome

Build the first end-to-end Tongues research proof that uses the new Conduit ML/dynamics substrate to learn a shared latent representation from real paired acoustic/articulatory speech data **without using phones, syllables, morae, word boundaries, or IPA segmentation as the training substrate**.

The purpose is not to claim a solved speech model. The purpose is to test the scientific hypothesis that a model forced to explain both acoustics and articulation can discover useful continuous latent dynamics from which traditional linguistic categories may later be recovered as projections/probes.

## Scientific background

Speech is not physically emitted as a string of perfectly bounded segments. Multiple articulators move on partially independent time courses, coordinated and correlated but not sharing one universal boundary clock. Tongue, jaw, lips, velum, laryngeal state, F0, etc. overlap and co-produce the acoustic result.

Tongues therefore should not begin by telling the model:

```text
/t/ = frames 91..104
/æ/ = frames 105..132
```

Instead it should begin with observed trajectories and sound, learn a dynamical latent account, and only afterward ask how that learned structure relates to linguistic annotation.

## Initial corpus

Choose a publicly usable paired acoustic/articulatory corpus with continuous measurements and sufficient documentation. Candidate families include XRMB/MNGU0/other EMA or X-ray microbeam corpora, subject to actual license/access and technical suitability.

The first proof should be deliberately small enough to run repeatably on the available std Host while retaining multiple speakers/examples where feasible.

Corpus ingestion must use #2139 semantics:

- exact corpus/manifest/resource identities;
- audio and articulatory source clocks;
- coordinate frames and physical units;
- calibration/head-correction transforms where supplied;
- missing-data masks;
- stable train/validation/test split;
- speaker/session identity required for evaluation while avoiding unnecessary personal metadata;
- preprocessing lineage.

### Acoustic side

Prefer a strong self-supervised speech representation as the initial front end rather than forcing the experiment to relearn basic acoustic invariances from a tiny articulatory corpus.

Candidates include wav2vec 2.0, HuBERT, WavLM or an equivalent open model that can be admitted as a #2137 ModelArtifact.

Important experimental questions:

- raw waveform vs SSL feature input;
- which SSL layer(s) best predict/articulate motion;
- whether the SSL front end is frozen, lightly adapted, or jointly fine-tuned;
- exact model/checkpoint identity and preprocessing provenance.

The front end is part of the experiment, not semantic truth.

### Articulatory side

Use synchronized/calibrated continuous coordinate trajectories such as tongue-tip/body/dorsum, lips, jaw, etc. represented through #2135.

Do not discretize them into place/manner labels before representation learning.

## Model architecture

Implement a first **bidirectional multimodal latent dynamical model**.

At minimum:

```text
acoustic observation -> acoustic encoder -> q(Z | X)
articulatory observation -> articulatory encoder -> q(Z | A)

Z -> acoustic decoder -> X_hat
Z -> articulatory decoder -> A_hat
```

With relation queries (#2142), the same learned system should support at least:

- audio -> latent posterior;
- articulation -> latent posterior;
- audio -> articulatory posterior/samples;
- articulation -> reconstructed/generated acoustics;
- latent -> both modalities.

## Continuous latent structure

The primary representation should remain continuous.

Investigate a latent time substrate that can support trajectories with their own learned dynamics, potentially using:

- continuous-time latent state / Neural ODE-style model;
- state-space model;
- recurrent latent process;
- graph/coupled latent processes;
- another architecture consistent with #2141.

Do not require one latent vector per fixed acoustic frame to be the final ontology even if a sampled implementation is used initially.

## Training objectives

Use #2140 to define a bounded objective profile including appropriate subsets of:

### Same-modality reconstruction

```text
X -> Z_X -> X_hat
A -> Z_A -> A_hat
```

### Cross-modal reconstruction

```text
X -> Z_X -> A_hat
A -> Z_A -> X_hat
```

### Paired latent agreement

For synchronized paired examples, encourage the acoustic and articulatory encoders to infer compatible distributions/trajectories over shared `Z`.

Do not require literal pointwise identity if time resolution/uncertainty makes a distributional or aligned objective more appropriate.

### Probabilistic regularization

Use VAE/flow/etc. regularization only as justified by the selected architecture; preserve exact objective identity in provenance.

### Cycle/consistency terms

Use cautiously.

Because acoustic -> articulation is underdetermined, do **not** require exact recovery of the original physical articulation after passing through acoustics if a different articulatory trajectory is acoustically/plausibly equivalent.

Prefer likelihood/plausibility, acoustic consistency, or posterior coverage metrics where scientifically appropriate.

## Probability and inversion

This proof must demonstrate that inverse inference can return uncertainty/multiple plausible trajectories through #2138.

For an acoustic observation, produce something like:

- posterior mean articulatory trajectory + variance; and/or
- several conditional articulatory samples;
- log probability/score where supported.

Do not present one decoded path as ground truth merely because it is the mean/MAP/sample.

## Speaker/anatomy/style factors

The model should leave room to factor sources of variation such as:

```text
z_g(t) gestural/vocal-tract dynamics
z_l(t) laryngeal/phonation dynamics
z_p(t) prosodic dynamics
s      speaker/anatomy
v      voice/source characteristics
r      rate/register/style
```

The first proof does not need perfect disentanglement, but it should avoid forcing all speaker identity into the same latent coordinates that are later interpreted as gestures.

At minimum compare speaker-conditioned vs speaker-agnostic variants or include a finite nuisance/context factor.

## Evaluation before linguistic labels

Before consulting phone annotation, evaluate whether the model actually explains the physical observations:

- held-out acoustic reconstruction/generation quality;
- held-out articulatory trajectory prediction;
- cross-modal prediction;
- posterior calibration/coverage where available;
- speaker-generalization or speaker-held-out performance where corpus size permits;
- stability under different seeds;
- whether generated articulation re-synthesizes compatible acoustics where a suitable synthesizer/decoder is available.

Do not reduce success to classification accuracy.

## Linguistic annotation is introduced afterward

Only after the representation is trained/frozen for a run should available phone/syllable/stress/word annotations be overlaid for analysis.

Questions include:

- Do known phone boundaries correspond to any consistent latent events?
- Are some latent variables more aligned with place, manner, voicing, vowel quality, stress, tone/F0, etc.?
- Are learned events systematically shorter/longer than traditional phones?
- Are gestural overlaps visible across conventional boundaries?
- Does onset/coda organization appear as temporal coupling rather than categorical boxes?

Annotations are probes, not the target used to create the latent representation.

## Discrete units: defer until after continuous learning

Do not put a VQ/discrete bottleneck at the deepest representation initially.

After a continuous latent model is working, run a controlled follow-up experiment adding clustering/VQ/category induction to ask:

> Where, if anywhere, does the continuous dynamical space naturally support stable recurring categories?

Possible discovered units may correspond to gesture onset/target/release, acoustic events, phone-like units, sub-phone events, or something else.

Failure to rediscover phones is scientifically informative and must not be repaired by silently training on phone labels.

## Patchbay proof

Use #2144 to make the experiment visually inspectable while running.

A capstone demo should show, at minimum:

- incoming/recorded audio waveform or acoustic feature trajectory;
- observed EMA/articulatory coordinates;
- inferred latent trajectories;
- inferred articulatory mean/uncertainty or multiple samples;
- loss/objective trajectories during a bounded training run;
- model-state/checkpoint generations;
- data moving through the actual Conduit Form.

The visualization should make temporal overlap evident rather than forcing vertical phone boundaries.

## Reproducibility

Commit experiment configuration/manifests sufficient to reproduce a bounded run, including:

- corpus/split manifest identity;
- preprocessing pipeline identities;
- pretrained acoustic model/checkpoint identity;
- Tongues model architecture/profile;
- objective weights;
- seed;
- training steps/work bounds;
- runtime/provider versions;
- produced checkpoint identity;
- evaluation metrics.

Large corpus/model artifacts should remain resource-referenced rather than checked into Git.

## Deliverables

- one or more `.conduit` Forms representing training and bidirectional inference/evaluation;
- dataset/corpus manifest/profile;
- std Host model adapter configuration;
- model/training checkpoint provenance;
- evaluation report generated from exact run evidence;
- Patchbay demo/proof;
- documentation explaining what was learned and what remains uncertain.

## Non-goals

- no claim of human-equivalent speech production/perception;
- no full ASR system requirement;
- no text/phoneme labels as mandatory training targets;
- no premature universal theory of phonology;
- no assumption one latent dimension corresponds to one anatomical articulator;
- no assumption the inverse articulation is unique;
- no requirement that the first model outperform specialist state-of-the-art AAI systems;
- no hiding failed hypotheses or negative results.

## Acceptance criteria

- [ ] Real paired audio/articulatory corpus is represented through #2139 with explicit clocks/calibration/masks/splits.
- [ ] Training representation does not consume phone/syllable/IPA boundaries or labels.
- [ ] Acoustic and articulatory encoders infer a shared latent process.
- [ ] Same-modality and cross-modal decoding work on held-out examples.
- [ ] Acoustic -> articulation returns probabilistic/alternative results rather than an unlabeled claim of unique truth.
- [ ] The same learned artifact/system supports queries in both principal directions through #2142.
- [ ] At least one continuous/recurrent dynamics realization is evaluated.
- [ ] Training is bounded, checkpointed, and reproducibly identified through Conduit.
- [ ] Patchbay can watch physical observations, latent trajectories, uncertainty, losses, and state transitions during a real run.
- [ ] Traditional linguistic annotation is introduced only after representation learning for probing/analysis.
- [ ] Results document both successful correspondences and failures/non-correspondences with conventional segment categories.

## Scientific test

The point of the experiment is to make this question empirical:

> **If a learning system must explain both the sound and the moving vocal tract, but is never told where the phonemes are, what temporal and categorical structure does it discover for itself?**

**Comment by dancxjo** [comment]
Starting from exact `main` commit `0f21edf5a876733364f2f4e274a7fdeb270e6510`.

Selected corpus: PB2007, Zenodo DOI `10.5281/zenodo.6390598`, exact archive SHA-256 `123d3fc2f114ab37724c7f05e00a03ff21d7e815f7f957987e8255f56d73f243`. The deposit is directly downloadable and contains 1,110 paired 16 kHz audio / 100 Hz twelve-coordinate EMA utterances. The deposited README states CC-BY-SA; Zenodo metadata currently labels it CC-BY-4.0. The experiment will retain that discrepancy explicitly and apply the more restrictive README statement to the derived slice.

Outcome: a small deterministic real-data slice, label-free bounded shared-latent training and bidirectional held-out queries through the existing #2133–#2144 contracts, exact checkpoint/evaluation evidence, post-freeze label probe, and real Patchbay projection. The run will enter through `cargo xtask` and remain small enough for PR CI and a browser-hostable report artifact.

Non-goals / stop line: no state-of-the-art claim, no hidden download in ordinary CI, no phonetic labels in representation training, no claim of cross-speaker generalization from this single-speaker corpus, no unique-truth label on inverse articulation, and no second runtime or Patchbay.

Proof needed: derivation reproducibility against the exact upstream archive; split leakage and label-quarantine refusals; deterministic bounded training/checkpoint identity; held-out same/cross-modal metrics and probabilistic inverse alternatives; post-freeze probe with successes and failures; exact-head CI; then fresh exact-main evidence before closure.

**Comment by dancxjo** [comment]
Implementation is in PR #2213 at exact head `d28c8a4f0d34eecdce5c60b96bc58808012d9bc8`. Waiting for required exact-head CI before merge; #2145 will remain open until fresh exact-main evidence is recorded.

**Comment by dancxjo** [comment]
PR #2213 head advanced to `4cbdb863305b6f5f51f0c1c7d3bcb0ae4592e9c6` with a CI-impact repair: manifest-backed `Cargo.lock` changes now follow package dependency closure, while unexplained lock-only changes retain the full-platform fallback. Exact-head checks restarted.

**Comment by dancxjo** [comment]
Accepted at exact main `f989cbba1803a88c959eecfd973bded33a84c039` after merge of #2213.

Evidence:
- Exact PR head `2825c6d303973ab39f60cb5211b8123fbf1a4c5c` passed required `check`: https://github.com/dancxjo/conduit/actions/runs/33709171580
- The same head passed the focused real-Patchbay browser proof, 5/5 with one Chromium worker and zero retries: https://github.com/dancxjo/conduit/actions/runs/33709171592
- Fresh exact-main rebuild and proof passed locally: `cargo clean -p patchbay-html`; `cargo +stable build --locked -p patchbay-html`; `cargo +stable test --locked -p conduit-tongues --test research_capstone` (3/3); focused Playwright proof (5/5).
- #2227 was closed unmerged after its clean CI run proved that an earlier local failure was a stale generated-product artifact, not a main regression.

The accepted slice uses PB2007 DOI `10.5281/zenodo.6390598`, source SHA-256 `123d3fc2f114ab37724c7f05e00a03ff21d7e815f7f957987e8255f56d73f243`, derived-slice SHA-256 `bedb30727379c3a46aea5cbcb34463caa569c4693f775d10479c77b0ed4c79a1`, an exactly identified bounded checkpoint, label-free training followed by frozen post-hoc probes, bidirectional relation signatures, probabilistic acoustic-to-articulation alternatives, and the authoritative Patchbay Watch.

The report remains explicit about the negative limits: one speaker, handcrafted acoustic features rather than SSL, unspecified head correction, tiny held-out sets, no calibrated-posterior, phone-class, causal, speaker-generalization, or state-of-the-art claim. Those constraints are not promoted into stronger scientific proof.
